db/projects/ncash/project.yaml

```diff
@@ -0,0 +1,136 @@
+    webids:
+      linkedin: dudleymartin
+  - name: Brooks Atwood
+    role: Head of Design, CCO
+    photo: https://cdn-images-1.medium.com/max/1600/1*feC45lC4_XKDAeOxcjVxZA.png
+    webids:
+      linkedin: brooksatwood
+webids:
+  email: info@nucleus.vision
+  linkedin/company: nucleusvision
+  facebook: NucleusVision
+  facebook/groups: '2044534525820774'
+  twitter: NucleusVision
+  github: NucleusVision
+  reddit: NucleusVision
+  youtube/channel: UCJ4LIGyMee0HnAx4rDvX0nw
+  medium: NucleusVision
+  crunchbase/person: nucleus-2e55
+  coinmarketcap: nucleus-vision
+  cryptocompare: ncash
+  bitcointalk: "2455575.0"
+  bitcoinwiki: Nucleus_Vision
+whitepapers:
+  - title: 'One Page Whitepaper'
+    url: https://nucleus.vision/onepager.pdf?v=3.0.1
+  - title: 'Light paper'
+    url: https://nucleus.vision/light-paper.pdf?v=3.0.1
+  - title: 'Whitepaper'
+    url: https://docsend.com/view/sfdfg6p
+tags:
+  - retail
+  - iot
+partners:
+  - title: "Tim Draper"
+    url: https://www.linkedin.com/in/timothydraper/
+  - title: "Alphabit Fund"
+    url: https://www.alphabit.fund/
+  - title: "Badwatel Capital"
+    url: https://www.badwater.capital/
+  - title: "T1R"
+    url: https://www.linkedin.com/company/t1rcse/
+  - title: "Wolf Fund"
+    url: https://www.wolf-fund.com/
+  - title: "CoinHako"
+    url: https://www.coinhako.com/
+  - title: "ICRISAT"
+    url: https://www.icrisat.org/
+  - title: "Shoppers Stop"
+    url: https://www.shoppersstop.com/
+  - title: "Future Group"
+    url: http://www.futuregroup.in/
+  - title: "Retailers Association of India"
+    url: http://rai.net.in/
+  - title: "Globus Stores Pvt Ltd"
+    url: https://www.globusfashion.com/
+  - title: "Host.Games"
+    url: https://www.host.games/
+  - title: "Croma"
+    url: https://www.croma.com/
+  - title: "Hashtag Loyalty"
+    url: https://www.hashtagloyalty.com/
+  - title: "Tech Mahindra"
+    url: https://www.techmahindra.com/DAVID.html
+  - title: "Eleven01"
+    url: https://www.eleven01.io/
+history:
+  - date_approx: "2013"
+    description: "First mention of Nucleus Vision"
+    title: "Begin of research (400 surveys)."
+    url: https://en.bitcoinwiki.org/wiki/Nucleus_Vision
+  - date_approx: "2014"
+    title: "Tim draper invested. Agreements with 100 trademarks and 4 Telecom companies."
+  - date_approx: "2015"
+    title: "Demo version tested in San Francisco."
+  - date_approx: "2016"
+    title: "Project completed, 4 global partnerships and 50M users registred trought telecom companies."
+  - date_approx: "2017"
+    title: "Launch of Nucleus Vision, 10 sensors installed, developement of coupoun system."
+  - date_approx: "2017-Q3"
+    title: "15 sensors, coupon system started to be used. 40k active users"
+resources:
+  - title: "The ION ease"
+    url: https://nucleus.vision/product
+  - title: "About Ncash technology - ORBIT & NEURON"
+    url: https://nucleus.vision/technology
+contacts:
+    email: "info@nucleus.vision"
```

Comment:
Already mentioned in webids

Reply:
I think its OK for both categories - it is needed in webids as a core information, but contact could be different (but it is not in this case)